src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -427,11 +427,12 @@ def load_command_table(self, _):
 
     with self.command_group('staticwebapp', custom_command_type=staticsite_sdk, is_preview=True) as g:
         g.custom_command('list', 'list_staticsites')
-        g.custom_command('browse', 'show_staticsite')
+        g.custom_show_command('show', 'show_staticsite')
         g.custom_command('create', 'create_staticsites', supports_no_wait=True)
         g.custom_command('delete', 'delete_staticsite', supports_no_wait=True, confirmation=True)
         g.custom_command('disconnect', 'disconnect_staticsite', supports_no_wait=True)
         g.custom_command('reconnect', 'reconnect_staticsite', supports_no_wait=True)
+        g.custom_command('update', 'update_staticsite', supports_no_wait=True)
```

Comment:
consider using https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#generic-update-commands for update command

Reply:
> 
> 
> consider using https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#generic-update-commands for update command

She doesn't use PUT method